src/Features/CSharp/Portable/CodeCleanup/CSharpCodeCleanupService.cs

```diff
@@ -178,6 +188,14 @@ internal class CSharpCodeCleanupService : ICodeCleanupService
 
                 // Mark this option as being completed.
                 progressTracker.ItemCompleted();
+
+                using (Logger.LogBlock(FunctionId.CodeCleanup_Format, cancellationToken))
```

Comment:
I meant adding this (https://github.com/chborl/roslyn/pull/1#pullrequestreview-136861042) back.

Reply:
OK, I think I have it in the right place now.